web/locale/en_US/messages.po

```diff
@@ -2696,17 +2696,23 @@ msgstr "No invites found"
+msgstr "Volunteers are the driving force behind Dojos around the world. Volunteers do everything from setting up the space, to mentoring during the Dojo, and to ensuring there is cool content for the Ninjas to do at each Dojo! Volunteers should join their Dojo on Zen to get updates on cool content, contribute to the CoderDojo Forums and to help administer the Dojo by awarding badges and setting up events!"
 
-msgid "Youth Over 13 - A Ninja is a young person who attends their local Dojo! Only Ninjas over 13 can have their own account, Ninjas under 13 must register under their Parent/Guardians account."
-msgstr "Youth Over 13 - A Ninja is a young person who attends their local Dojo! Only Ninjas over 13 can have their own account, Ninjas under 13 must register under their Parent/Guardians account."
+msgid "Attendees at CoderDojo are young people under 18 who attend and make CoderDojo Cool."
+msgstr "Attendees at CoderDojo are young people under 18 who attend and make CoderDojo Cool."
 
-msgid "Parent/Guardian - Parents/kids are super important in CoderDojo as they encourage and inspire CoderDojo Ninjas on a daily basis! Parents can sign up on the platform to register their kids (both aged under 13 and over 13) for their Local Dojos events and so their kids can earn badges for their profile!"
-msgstr "Parent/Guardian - Parents/kids are super important in CoderDojo as they encourage and inspire CoderDojo Ninjas on a daily basis! Parents can sign up on the platform to register their kids (both aged under 13 and over 13) for their Local Dojos events and so their kids can earn badges for their profile!"
+msgid "Parents/Guardians are super important in the CoderDojo journey as they encourage and inspire CoderDojo Ninjas on a daily basis. Parents should register and join a Dojo so the young people attached to their account can sign up to events and earn badges from their Champions and Mentors."
+msgstr "Parents/Guardians are super important in the CoderDojo journey as they encourage and inspire CoderDojo Ninjas on a daily basis. Parents should register and join a Dojo so the young people attached to their account can sign up to events and earn badges from their Champions and Mentors."
+
+msgid "You will be joined to a Dojo as an adult, along with any young people added to your profile. We would like to know if you have any volunteering role in the Dojo too."
+msgstr "You will be joined to a Dojo as an adult, along with any young people added to your profile. We would like to know if you have any volunteering role in the Dojo too."
+
+msgid "If you join the Dojo you will receive event updates from the Dojo and some newsletters from CoderDojo HQ! If you aren't a Dojo attendee and want to join as a volunteer or Champion please get in touch with info@coderdojo.org"
```

Comment:
Kids get specific newsletter (still have to have checked box).

either email works - use whichever you prefer. This text is not for Adults but a continuation of the Ninja 'pop-up' informing them why to join a Dojo. If they are accidentally signed up as youth they could get in contact with us but yes on second read it isn't the best.

Replace with;
"If you join the Dojo you will receive event updates from the Dojo and some newsletters from CoderDojo HQ! If you aren't a Dojo attendee and want to join as a volunteer or Champion please get in touch with the Dojo directly."